brands/tourism/hotel.json

```diff
@@ -520,6 +520,18 @@
       "tourism": "hotel"
     }
   },
+  "tourism/hotel|TownePlace Suites by Marriott": {
```

Comment:
I think we want to just make this TownePlace Suites and then have a matchname for TownePlace Suites by Marriot and TownePlace Suites Marriot.

Reply:
I changed the name and added "TownePlace Suites Marriott" and "TownePlace Marriott" as match names.  "TownePlace Suites by Marriott" will be matched via `official_name`